src/frontend/src/lib/i18n/en.json

```diff
@@ -125,7 +125,7 @@
 	},
 	"hero": {
 		"text": {
-			"use_with_caution": "Learn more about this alpha version. Use with caution.",
+			"use_with_caution": "This is alpha version. Use with caution.",
```

Comment:
Isn't a "an" missing in the sentence? "This is an alpha" version or something?

Reply:
I think so to, and in fact I wanted to have even "This is THE Alpha version..."; let me ask Max (native speaker eheh)

Reply:
Checked with Max: as you suggested, `This is an alpha version...` is the better version